src/word-search/index.js

```diff
@@ -1,3 +1,189 @@
-function wordSearch(words, word) {}
+function wordSearch(words, word) {
```

Comment:
format your code properly, remove the unnecessary white spaces

Reply:
aiit i would do just that. thanks

Reply:
i have done that .